test/server/cards/06-WoE/MassBuyout.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('Mass Buyout', function () {
```

Comment:
Can you add test cases for warded creatures and include creatures that goes to other places other than discard, such as Bad Penny? I think it might fail with warded creatures.

Reply:
The test already has a warded creaure (`Gub`).  I can add Bad Penny in later, but this code is just copied directly from Final Analysis so it should have the same behavior as that one.

Reply:
(The ward is what makes player2 in the second test only gain 1 amber, rather than 2).